tests/Dockerfile.vic-integration-test-base

```diff
@@ -0,0 +1,30 @@
+# VIC Test Image
+#
+# Build a docker image that runs latest Ubuntu with golang, robotframework and vmware-vic installed
+# (https://golang.org/)
+# (http://robotframework.org/)
+# (https://github.com/vmware/vic)
+#
+# docker build -t vic-integration-test-base -f Dockerfile.vic-integration-test-base .
+
+from ubuntu:16.04
+
+# Make sure image is up to date and required libraries are installed
+run apt-get update
+run apt-get -y upgrade
+run apt-get -y install python-pip
+run apt-get -y install wget
+run apt-get -y install git
+run apt-get -y install build-essential
+run apt-get -y install docker.io
+run apt-get -y install iputils-ping
+run pip install --upgrade pip
+run pip install robotframework
+run pip install requests
+
+# Install version 1.6.2 of golang
+run wget https://storage.googleapis.com/golang/go1.6.2.linux-amd64.tar.gz
+run tar xvf go1.6.2.linux-amd64.tar.gz
```

Comment:
There is a go1.6.2 docker image you can inherit from.  It's based on debian jessie unless you absolutely require ubuntu.